editor/components/block-edit/context.js

```diff
@@ -59,3 +60,44 @@ export const ifBlockEditSelected = createHigherOrderComponent( ( OriginalCompone
 		</Consumer>
 	);
 }, 'ifBlockEditSelected' );
+
+/**
+ * A Higher Order Component used to render conditionally the wrapped
+ * component only when the BlockEdit has selected state set or it is
+ * the first block in a multi selection of all one type of block..
+ *
+ * @param {Component} OriginalComponent Component to wrap.
+ *
+ * @return {Component} Component which renders only when the BlockEdit is selected or it is the first block in a multi selection.
+ */
+
```

Comment:
This newline shouldn't be here, at risk of the JSDoc not being associated with the function.